### DISCUSSION TITLE
Question on reliability

### DISCUSSION THREAD

**Discussion by rgviewer** [gtxaspec/wz_mini_hacks#743] [General]
I am wondering if most people find using wz_mini_hacks find it to be pretty reliable. Right now I have 4 wyze v2's using docker wyze bridge sending rtsp to Frigate. For the most part they work pretty good, I do occasionally get camera's loose connectivity and have to power cycle cams to get stream back up. Also I see a lot of "Unable to read frames from ffmpeg process" in the logs of Frigate. Wondering if I would be better to give wz_mini_hacks a go. 
One other question for anyone, do most people self-host their wyze cams?
Thnanks

**Comment by digitalpixies** [comment]
I get the "Unable to read frames" problem as well but only from my wifi connected v3. If you're interested in reliable connection, the best option I took was to use "USB CDC_NCM" support from wz_mini_hacks and make the v3s all communicate via usb (wifi is no longer used). In my situation, I have a rpi4 connected to 1 or 2 of these v3. You have to be familar with setting up peer to peer networks since USB CDC_NCM makes the rpi4 see the v3's as tcp peer networking clients. rpi4 seems to run frigate fine but I've put one of them under heavy strain/load and it's overheating now and not running reliably. Another one I setup has been running fine so far (although less motion is triggered which lowers frigate cpu use). If you use a N100 mini pc I believe the setup will be fine.

I have had to power cycle once or twice a year on my wifi connected one when it looses connectivity. On some occasions when the v3 doesn't respond, I have to re-flash wz_mini_hacks onto the v3.

**Reply by rgviewer** [reply]
Thanks for getting back, I did end up putting wyze-mini hacks on my V2 cams, I still get errors in log like Non-monotonous DTS in output stream 0:0; previous: 46115806, current: 46115806; changing to 46115807. This may result in incorrect timestamps in the output file.2024-03-26 12:34:38.171781233  [2024-03-26 12:34:38] ffmpeg.porch_cam.record,   ERROR   : rtsp://127.0.0.1:8554/porch_cam: Connection timed out. Also Ffmpeg process crashed unexpectedly for porch_cam.

I am running Frigate in LXC within Proxmox, with USB TPU. I am thinking it may be a wifi problem and may want to look at adding POE adapter, or just getting better camera's. I have a Reolink RLC-810A that I got for Christmas but have not hooked up yet. Been reading poeple have issues with Reolink as well though. 

My quest continues to have a fully working, error free setup if that is even possible. 
Thanks again
